Search/Converter/Types/DateConverter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Converter\Types;
+
+use Massive\Bundle\SearchBundle\Search\Converter\ConverterInterface;
+
+class DateConverter implements ConverterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function convert($value)
+    {
+        return is_null($value) ? '' : $value->format('Y-m-d\TH:i:sP');
```

Comment:
The format depends on the used adapter, so I wouldn't make that configureable. Although it makes sense to extract the value for sure (there are already some constants for date formats included, maybe we can even use these).